examples/neural_networks/plot_rbm_logistic_classification.py

```diff
@@ -79,7 +79,7 @@ def shift(x, w):
 X, y = datasets.load_digits(return_X_y=True)
 X = np.asarray(X, 'float32')
 X, Y = nudge_dataset(X, y)
-X = (X - np.min(X, 0)) / (np.max(X, 0) + 0.0001)  # 0-1 scaling
+X = (X - np.min(X, 0)) / (np.max(X, 0) - np.min(X, 0))  # 0-1 scaling
```

Comment:
[As proposed by @amueller](https://github.com/scikit-learn/scikit-learn/issues/19362#issuecomment-774400231):
```suggestion
X = minmax_scale(X, feature_range=(0, 1))  # 0-1 scaling
```

Reply:
Thank you!